lib/plant_fba/core/fetch_plantseed_impl.py

```diff
@@ -0,0 +1,81 @@
+from urllib.request import urlopen
+import time
+import json
+import os
+import re
+
+PS_url = 'https://raw.githubusercontent.com/ModelSEED/PlantSEED/'
```

Comment:
@samseaver I would like to push back on this.  I would prefer that we fetch this during registration.  We don't want these potentially changing underneath us.

Reply:
So what I suggest is fetch these in the Dockerfile and then directly open this instead of fetching it at runtime from github.